### PR TITLE
Lab5

### LINKED ISSUE #570: Lab 5 #527
Fix #527

### PR DESCRIPTION
**Pull request by hsy3418**
Fixes #570 

Review of colleague's PR #291

#### Changes proposed in this PR:

- Create a method to print the IP address as a dotted squad

### CODE REVIEW

**Comment by patrickmarabeas** [comment]
Code looks good. 

You've seemed to have included your numeronym commit in this PR, you should clean up your commit history and re push.

PR needs to be updated to include the issue # it fixes; the reviewed PR #; description and formatting.

**Review by juliaogris — CHANGES_REQUESTED** [review]
Please fix PR description, add link to issue and review of colleague's PR.
Other than that: LGTM.

File: 05_stringer/hsy3418/main_test.go [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+}
+
+func TestMainOutput(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+
+	main()
+
+	expected := strconv.Quote("127.0.0.1\n")
+	actual := strconv.Quote(buf.String())
```
Comment (juliaogris) [inline 05_stringer/hsy3418/main_test.go:38]: remove `Quote` from both

File: 05_stringer/hsy3418/main_test.go [hunk 2]
```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"testing"
+)
+
+func TestStringer(t *testing.T) {
+
```
Comment (juliaogris) [inline 05_stringer/hsy3418/main_test.go:10]: unnecessary blank line

**Review by juliaogris — APPROVED** [review]
LGTM, just a few minor nitpicky comments - feel free to ignore.

**Pull request merged**